suite-native/state/src/StoreProvider.tsx

```diff
@@ -18,10 +19,15 @@ export const StoreProvider = ({ children }: StoreProviderProps) => {
 
     useEffect(() => {
         const initStoreAsync = async () => {
-            const freshStore = await initStore();
-            const freshPersistor = persistStore(freshStore);
-            setStore(freshStore);
-            setStorePersistor(freshPersistor);
+            try {
+                const freshStore = await initStore();
+                const freshPersistor = persistStore(freshStore);
+                setStore(freshStore);
+                setStorePersistor(freshPersistor);
+            } catch (error) {
+                console.error('Init store error:', error);
+                Sentry.captureException(new Error(error));
```

Comment:
I think you can just do `Sentry.captureException(error as Error);`

Reply:
It's already with correct type, my bad
https://github.com/trezor/trezor-suite/pull/12928/commits/01ec4fbe77a6845bb528fbc819a1ef5bfbbbdca6